test/libsinsp_e2e/event_capture.cpp

```diff
@@ -170,6 +170,7 @@ void event_capture::capture()
 		m_inspector->stop_capture();
 
 		delete m_inspector;
+		m_inspector = nullptr;
```

Comment:
This is a fix so that when a test fails, tests don't crash.

Reply:
This will conflict with changes from https://github.com/falcosecurity/libs/pull/1754. Will fix the conflict once that is merged.